plugin/output/kafka/kafka.go

```diff
@@ -162,11 +162,7 @@ func (p *Plugin) out(workerData *pipeline.WorkerData, batch *pipeline.Batch) {
 			data.messages[i] = &sarama.ProducerMessage{}
 		}
 		data.messages[i].Value = outBuf[start:]
-
-		// copy topic from json, to temporary out buffer to avoid event reusing issues
-		start = len(outBuf)
-		outBuf = append(outBuf, topic...)
-		data.messages[i].Topic = pipeline.ByteToStringUnsafe(outBuf[start:])
+		data.messages[i].Topic = pipeline.DeepCopyString(topic)
```

Comment:
Let's just copy string only at 157 ? What do you think about it ?

Reply:
Good point! 
Done